src/eXpansion/Bundle/CustomUi/Resources/config/plugins.yml

```diff
@@ -79,9 +79,8 @@ services:
             $uiProperties: "%expansion.custom_ui.ui_properties%"
             $setPropertiesScriptEvent: 'Trackmania.UI.SetProperties'
             $customWidgets:
-                - '@eXpansion\Bundle\CustomUi\Plugins\Gui\CustomSpeedWidget'
+#                - '@eXpansion\Bundle\CustomUi\Plugins\Gui\CustomSpeedWidget'
```

Comment:
If the class & service is not being used can we delete it instead of keeping unused stuff.